src/map_extras.cpp

```diff
@@ -146,15 +145,12 @@ static const map_extra_id map_extra_mx_supplydrop( "mx_supplydrop" );
 
 static const mongroup_id GROUP_FISH( "GROUP_FISH" );
 static const mongroup_id GROUP_FUNGI_FUNGALOID( "GROUP_FUNGI_FUNGALOID" );
-static const mongroup_id GROUP_MAYBE_MIL( "GROUP_MAYBE_MIL" );
```

Comment:
GROUP_MAYBE_MIL( "GROUP_MAYBE_MIL" ) is deleted here in the C++ but I don't see it in the JSON?   Is that because it was replaced by the nested chunks?